src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -657,7 +657,7 @@ public void TestRuby_PascalCaseNotAccepted()
         public void TestCSharp_LowerCaseAccepted()
         {
             Helper.AssertTemplateResult("", "{% if 'bob' startswith 'B' %} YES {% endif %}", null, new CSharpNamingConvention());
-            Helper.AssertTemplateResult(" YES ", "{% if 'Bob' startswith 'B' %} YES {% endif %}", null, new CSharpNamingConvention());
+            Helper. AssertTemplateResult(" YES ", "{% if 'Bob' startswith 'B' %} YES {% endif %}", null, new CSharpNamingConvention());

```

Comment:
Please revert this change. Space between a class name and method is atypical coding style.

Reply:
Fixed: https://github.com/dotliquid/dotliquid/pull/453/commits/42ea12d8664736c8ab5a3aeff8f9f62e3049fdd2